### ISSUE TITLE
add docker_service labels option

### ISSUE THREAD

**Issue by anvibo** [terraform-providers/terraform-provider-docker#132]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.11.11
+ provider.docker v1.1.0
+ provider.local v1.1.0

### Affected Resource(s)
Please list the resources as a list, for example:
- docker_service

### Terraform Configuration Files
```hcl
resource "docker_service" "traefik" {
    name = "traefik-service"

    task_spec {
        container_spec {
            image = "traefik"

            labels {
                traefik.frontend.rule = "Host:proxy.anvibo.com"
                traefik.port = 8080
            }

            configs = [
                {
                    config_id   = "${docker_config.traefik-toml.id}"
                    config_name = "${docker_config.traefik-toml.name}"
                    file_name = "/traefik.toml"
                },
            ]

            mounts = [
                {
                    target      = "/var/run/docker.sock"
                    source      = "/var/run/docker.sock"
                    type        = "bind"
                    read_only   = true
                },
            ]
        }
        networks     = ["${docker_network.proxy.id}"]
    }

    endpoint_spec {
      ports {
        target_port = "80"
        published_port = "80"
      }
      ports {
        target_port = "443"
        published_port = "443"
      }
    }
}
```

### Expected Behavior
I would like to have the possibility to add labels also on service level, not only on backing container
This is required in order to be able to deploy docker services that can be autodiscovered by traefik proxy

### Actual Behavior
Currently, I see no option to add labels on service level

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by andrejbaran** [comment]
@anvibo you need to specify `labels` section on the same level as `task_spec` like:
```terraform
resource "docker_service" "traefik" {
    name = "traefik-service"
  
    labels {
        traefik.frontend.rule = "Host:proxy.anvibo.com"
        traefik.port = 8080
    }

    task_spec {
        container_spec {
            image = "traefik"

            configs = [
                {
                    config_id   = "${docker_config.traefik-toml.id}"
                    config_name = "${docker_config.traefik-toml.name}"
                    file_name = "/traefik.toml"
                },
            ]

            mounts = [
                {
                    target      = "/var/run/docker.sock"
                    source      = "/var/run/docker.sock"
                    type        = "bind"
                    read_only   = true
                },
            ]
        }
        networks     = ["${docker_network.proxy.id}"]
    }

    endpoint_spec {
      ports {
        target_port = "80"
        published_port = "80"
      }
      ports {
        target_port = "443"
        published_port = "443"
      }
    }
}
```

This should be probably added to the docs.

**Comment by anvibo** [comment]
That seems to be not accepted 
Error: module.traefik.docker_service.traefik: task_spec.0: invalid or unknown key: labels

**Comment by andrejbaran** [comment]
@anvibo what version of the plugin do you use? it's been there since `v1.0.0`

**Comment by mavogel** [comment]
@andrejbaran he uses `provider.docker v1.1.0`

**Comment by mavogel** [comment]
Should be fixed with #214 . We restructured the label sections because of HCL2 Map syntax does not accept `.` within a key.
